jscomp/super_errors/super_typetexp.ml

```diff
@@ -159,7 +159,11 @@ let setup () =
   Location.register_error_of_exn
     (function
       | Typetexp.Error (loc, env, err) ->
-        Some (Location.error_of_printer loc (report_error env) err)
+        Some (Location.error_of_printer loc (fun ppf ->
```

Comment:
can you elaborate a little bit, I did not follow you. You know the input is reason file or not very early? (btw, we can add a flag like -bs-reason-error etc)